src/logging.ts

```diff
@@ -0,0 +1,7 @@
+export function log(msg: string) {
```

Comment:
Maybe `log` can be `debug`? Since it uses the `console.debug`?

Reply:
Changed to debug.